lab_1/main.py

```diff
@@ -3,81 +3,148 @@
 Language detection
 """
 
+import json
 
-def tokenize(text: str) -> list or None:
+
+def tokenize(text: str):
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation and other symbols from words
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
-    """
-    pass
+        """
+    if isinstance(text, str):
+        text = text.lower()
+        string_with_tokens = ""
+        for i in text:
+            if i.isalpha() or i.isspace():
+                string_with_tokens += i
+        tokens = string_with_tokens.split()
+        return tokens
+    return None
 
 
-def remove_stop_words(tokens: list, stop_words: list) -> list or None:
+def remove_stop_words(tokens: list, stop_words: list):
     """
     Removes stop words
     :param tokens: a list of tokens
     :param stop_words: a list of stop words
     :return: a list of tokens without stop words
     """
-    pass
+    try:
```

Comment:
wow, why do you need `try/except`? maybe a simple `if` will do the job?

Reply:
and as I said, checks with None to the beginning

Reply:
Yes, it works now without try/exept!
done